KsApi/models/GraphUserCreditCard.swift

```diff
@@ -6,18 +6,22 @@ public struct GraphUserCreditCard: Swift.Decodable {
     public private(set) var expirationDate: String
     public private(set) var id: String
     public private(set) var lastFour: String
-    public private(set) var type: CreditCardType
+    public private(set) var type: CreditCardType?
 
     public var formattedExpirationDate: String {
       return String(expirationDate.dropLast(3))
     }
 
     public var imageName: String {
-      switch self.type {
+      guard let cardType = self.type else {
```

Comment:
We should be able to use Swift's pattern matching in order to get rid of the `guard`.

I think this should work without the need to repeat `icon--generic` twice.

```
switch self.type {
  case nil, .some(.generic):
    return "icon--generic"
  case .some(let type):
    return "icon--\(type.rawValue.lowercased())"
}
```